src/main/java/com/plaid/client/request/InstitutionsGetByIdRequest.java

```diff
@@ -41,8 +41,34 @@ public InstitutionsGetByIdRequest withIncludeStatus(boolean includeStatus) {
     return this;
   }
 
+  public InstitutionsGetByIdRequest withIncludePaymentInitiationMetadata(boolean includePaymentInitiationMetadata) {
+    if (this.options == null) {
+      this.options = new Options();
+    }
+    this.options.includePaymentInitiationMetadata = includePaymentInitiationMetadata;
+    return this;
+  }
+
+  public InstitutionsGetByIdRequest withPaymentInitiation(PaymentInitiation paymentInitiation) {
```

Comment:
Oh oops forgot to call out that we don't do this for this endpoint either, the get by ID endpoint doesn't support filtering at all since it's specifically requesting a single institution.